charts/ziti-controller/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,61 @@
+{{- with .Values.prometheus }}
+{{- if and .service.enabled .serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "ziti-controller.fullname" $ }}
+  {{- with .serviceMonitor.namespace }}
+  namespace: {{ . }}
+  {{- end }}
+  {{- with .serviceMonitor.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  labels:
+    {{- include "ziti-controller.labels" $ | nindent 4 }}
+    {{- with .serviceMonitor.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  {{- with .serviceMonitor.namespaceSelector }}
```

Comment:
Shouldn't the namespace selector be the Helm release's namespace as opposed to an arbitrary input value?

Reply:
It is a little complicated, but the pattern seems to be to install all servicemonitors in your *monitoring* namespace,

Something like 
```yaml
namespace: monitoring
namespaceSelector:
  matchNames:
    - openziti
```